data_preparation/data_cleaning.py

```diff
@@ -1,41 +1,37 @@
 import sys
 import os
-
 module_path = os.path.abspath(os.path.join('..'))
-print(module_path)
-print(sys.path)
-if module_path not in sys.path:
-    sys.path.append(module_path)
+sys.path = [module_path] + sys.path
 
-# import mysql.connector
-# from data_preparation.autotrader_scraper.autotrader_scraper.config import mysql_details
+import mysql.connector
+from data_preparation.autotrader_scraper.autotrader_scraper.config import mysql_details
 import pandas as pd
 from sklearn.linear_model import LinearRegression
 import numpy as np
 from collections import defaultdict
 
-# def get_data_from_database() -> pd.DataFrame:
-#     '''
-#     Return full dataset with vehicle features and seller information joined
-#     in one table.
-#     '''
+def get_data_from_database() -> pd.DataFrame:
+    '''
+    Return full dataset with vehicle features and seller information joined
+    in one table.
+    '''
 
-#     DB_NAME = 'autotrader_adverts'
+    DB_NAME = 'autotrader_adverts'
     
-#     cnx = mysql.connector.connect(**mysql_details)
-#     cursor = cnx.cursor(dictionary=True)
-
-#     cursor.execute("USE {}".format(DB_NAME))
-#     cursor.execute('''SELECT * 
-#                   FROM vehicle_features as vf
-#                   LEFT JOIN sellers as s
-#                   ON vf.seller_id=s.seller_id
-#                   ORDER BY date_scraped ASC, time_scraped ASC''')
+    cnx = mysql.connector.connect(**mysql_details)
+    cursor = cnx.cursor(dictionary=True)
+
+    cursor.execute("USE {}".format(DB_NAME))
+    cursor.execute('''SELECT * 
+                  FROM vehicle_features as vf
+                  LEFT JOIN sellers as s
+                  ON vf.seller_id=s.seller_id
+                  ORDER BY date_scraped ASC, time_scraped ASC''')
```

Comment:
tbf you could leave this commented out permanently and we could just work from the csv file instead

Reply:
Hmmm okay, I guess we can setup the db stuff later on if you want it. Though the file sizes we're dealing with, don't think it's strictly needed, but still could be good practice setting it up.